crowdsourcing/serializers/user.py

```diff
@@ -135,15 +137,18 @@ def create(self, **kwargs):
         user_profile = models.UserProfile()
         user_profile.user = user
         user_profile.save()
-        worker = models.Worker()
-        worker.profile = user_profile
-        worker.alias = username
-        worker.save()
-
-        requester = models.Requester()
-        requester.profile = user_profile
-        requester.alias = username
-        requester.save()
+
+        if 'is_worker' not in self.validated_data or self.validated_data['is_worker']:
```

Comment:
gotcha, so by default neither will be created, it's a part of profile setup later (this was temporary for UIST)